cmd/offat/main.go

```diff
@@ -112,12 +117,21 @@ func main() {
 	parser.FuzzDocHttpParams()
 
 	// Create http client and config
-	httpCfg := http.NewConfig(config.RequestsPerSecond, config.SkipTlsVerification, config.Proxy)
-	hc := http.NewHttp(httpCfg)
+	var hc c.ClientInterface // changed hc to interface
+	if config.HTTP2 != nil && *config.HTTP2 {
+		hc = http.NewConfigHttp2(config.RequestsPerSecond, config.SkipTlsVerification, config.Proxy)
+	} else {
+		httpCfg := http.NewConfig(config.RequestsPerSecond, config.SkipTlsVerification, config.Proxy)
+		new_http := http.NewHttp(httpCfg)
+		hc = new_http.Client.FHClient
+	}
 
 	// Test server connectivity
-	url := *parser.Doc.GetBaseUrl()
-	resp, err := hc.Client.FHClient.Do(url, fasthttp.MethodGet, nil, nil, nil)
+	// url := *parser.Doc.GetBaseUrl()
+	url := "https://petstore.swagger.io/v2/store/inventory" // Uncomment for testing
```

Comment:
we can remove hardcoded values
```suggestion
	url := *parser.Doc.GetBaseUrl()
```